qemu/qemu.sh

```diff
@@ -51,7 +51,7 @@ append_args()
 	echo -n "debug "
     fi
 
-    echo -n "${QEMU_APPEND} ${QEMU_EXTRA_APPEND} "
+    echo -n "${QEMU_APPEND} bonding.max_bonds=0 dummy.numdummies=0 fb_tunnels=none ${QEMU_EXTRA_APPEND} "
```

Comment:
As agreed offline: Build these as modules and set these arguments in `/etc/modprobe.d/`. This way, we will get the same behavior in GNS3, on hardware, etc.

Reply:
Fixed in [a3195ef](https://github.com/kernelkit/infix/pull/13/commits/a3195ef6f3396db536ec172214e57f0268a13638)